Module2/question2.js

```diff
@@ -0,0 +1,45 @@
+// 2] Use let and const to create arrays and objects. Try modifying, deleting properties within the array or object. 
+//    What do you expect to happen in both cases? What actually happens in both cases. 
+//    What is the difference between an object declared as a let or a const variable?
+
+let myArr = [5,10,15]
+console.log(myArr)
```

Comment:
When you do console log...also add the result here in comments as // [5,10,15]

Reply:
Yes, I understand the mistake and also correct  it.